mne/beamformer/_compute_beamformer.py

```diff
@@ -301,10 +301,11 @@ def _compute_beamformer(G, Cm, reg, n_orient, weight_norm, pick_ori,
         The source orientation to compute the beamformer in.
     reduce_rank : bool
         Whether to reduce the rank by one during computation of the filter.
-    rank : int | 'auto' | None
-        The effective rank of the covariance matrix. If 'auto', it will be
-        estimated before regularization is applied. If ``None``, it will be
-        estimated after regularization is applied.
+    rank : None | False | int
```

Comment:
On second thought, `None`, `False`, `int` is very unintuitive. Also, it should default to `False` to keep the current behavior (this is probably why the example figure has changed)